src/transformers/models/mask2former/convert_mask2former_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -676,13 +853,13 @@ def get_name(checkpoint_file: Path):
 
     checkpoints_dir: Path = args.checkpoints_dir
     config_dir: Path = args.configs_dir
-    save_directory: Path = args.pytorch_dump_folder_path
+    save_directory: Path = args.save_model_dir
     mask2former_dir: Path = args.mask2former_dir
     # append the path to the parents to mask2former dir
     sys.path.append(str(mask2former_dir.parent))
-    # and import what's needed
-    from Mask2Former.mask_former import add_mask_former_config
-    from Mask2Former.mask_former.mask_former_model import Mask2Former as OriginalMask2Former
+    # import original Mask2Former config and model from original source code repo
+    from Mask2Former.mask2former.config import add_maskformer2_config
```

Comment:
Maybe I'm missing it but it seems the `add_maskformer2_config` function is not defined anywhere.

This function should basically map the configuration variable names and values of the original model to their HF implementation version.